.github/workflows/build.yml

```diff
@@ -69,8 +69,12 @@ jobs:
       - name: Deploy to S3 | Staging
         if: github.ref == 'refs/heads/staging'
         run: aws s3 sync ./public/ s3://cru-givestage --delete
-
-      - name: Deploy to S3 | Production
+      
+      - name: First Deploy to S3 | Production
+        if: github.ref == 'refs/heads/master'
+        run: aws s3 sync ./public/ s3://cru-giveprod
+      
+      - name: Second Deploy to S3 | Production
```

Comment:
This code looks good but idk how self explanatory it is. Maybe adjust the name or add a comment to make it clear that we're making sure the new files are all available before starting to delete any.

Reply:
Also let's do the Cloudfront invalidation before deleting any files on production. Idk if the CLI waits for the invalidation to finish but might be better to start it sooner. We don't want a file cached by Cloudfront referencing an uncached but deleted file.

As you can see this Webpack build/caching strategy could use some cleanup. Normally you cache `index.html` for a short time (say 3-5 mins) and then give every other file a unique hash in the filename and cache them for a year or more cuz if there are changes, they will end up with a new filename. I've setup apps so there's one main js/json file that loads everything else similar to how index.html would but this app hasn't been setup that way yet. Brian improved it at one point but there's still work to do as I don't think it's safe to cache everything for a year yet.